R/functions-qwtimeseries.R

```diff
@@ -63,14 +64,14 @@ makeTimeseriesPlot <- function(parm_data, y_axis_name, date_info, isTrophicIndex
                              plot_main_title, plot_sub_title) %>% 
```

Comment:
the reason you can't get `mar` to work is because `plotSetup` is called twice in the body of this function. You added `top_margin` and `bottom_margin` as arguments to the next one but that's the code only affecting the TSI plot. This line here (63-64) is the the `plotSetup` call for the other 3 plots. I'd suggest reverting your `oma` change and try your value for `top_margin` again once you add those arguments to this line!

Reply:
OK. Got it!  And it worked - the main title now appears!  But....the plots are all different sizes. :P  Next mission...

Reply:
I almost wonder if that's possible ... the plots are different sizes because their margins are different sizes. This was a side effect I did not consider. We might just need to make all the margins the same.

Reply:
OR increasing the top & bottom `oma` along with `mar` might get you to the right sizes....I'm not sure.

Reply:
I am exploring options...this one looks complicated, but you'd know better if it's even feasible:
http://seananderson.ca/courses/11-multipanel/multipanel.pdf
Pages 7-11 talk about creating pretty complicated layouts, one idea of which was using a matrix to create margins....Thoughts?